packages/xrpl/src/models/transactions/accountDelete.ts

```diff
@@ -28,6 +28,11 @@ export interface AccountDelete extends BaseTransaction {
    * information for the recipient of the deleted account's leftover XRP.
    */
   DestinationTag?: number
+  /** Credentials associated with sender of this transaction. The credentials included
+   * must not be expired. The list must not be empty when specified and cannot contain
+   * more than 8 credentials.
+   */
+  CredentialIDs?: string[]
 }
 
 /**
```

Comment:
@achowdhry-ripple  Validations associated with `CredentialDs` field is missing for this transaction.

Reply:
good catch, added a call to the helper function